pkg/templates/charts/toggle/cluster-backup/templates/hub-backup-pod.yaml

```diff
@@ -58,6 +61,11 @@ spec:
                   namespace: open-cluster-management-backup
                 status:
                   phase: Running
+          customMessage:
+            noncompliant: |-
+              Template is not compliant, openshift-adp-controller-manager pod is either not running in the open-cluster-management-backup namespace or is in error. 
+              If the openshift-adp-controller-manager pod is not running in the open-cluster-management-backup namespace check the 
+              multiclusterhub-operator pod for errors related to the OADP operator installation.
```

Comment:
Is checking the MultiClusterHub resource not good enough for this?  Just wondering if there is a simpler procedure for the message.

Reply:
@gparvin if you have an issue with the OADP channel that would show up in the MCH logs

Reply:
done, as discussed, replaced **Template is not compliant**, with **The**